### ISSUE TITLE
Weird "directrory does not exist" error

### ISSUE THREAD

**Issue by kmgalanakis** [coenjacobs/mozart#20]
Hi!

I'm trying to use mozart in Windows 10 and I'm having the following `composer.json` file:

```
{
    "name": "kmgal/mozart-test",
    "authors": [
        {
            "name": "Konstantinos Galanakis",
            "email": "kmgalanakis@gmail.com"
        }
    ],
    "require": {},
	"require-dev": {
        "coenjacobs/mozart": "^0.3.0",
        "paypal/rest-api-sdk-php": "*"
    },
	"scripts": {
      "post-install-cmd": [
        "\"vendor/bin/mozart\" compose",
        "composer dump-autoload"
      ],
      "post-update-cmd": [
        "\"vendor/bin/mozart\" compose",
        "composer dump-autoload"
      ]
    },
    "extra": {
      "mozart": {
        "dep_namespace": "MozartTest\\",
        "dep_directory": "/src/",
        "classmap_directory": "/classes/",
        "classmap_prefix": "MozartTest_",
        "packages": [
          "paypal/rest-api-sdk-php"
        ]
      }
    }
}
```

When running composer install, everything seems to install fine but at the moment `mozart compose` runs, I get this error

```
In Finder.php line 547:

  The "C:\Users\kmgal\Desktop\mozart-test/src/PayPal/" directory does not exist.


compose

Script "vendor/bin/mozart" compose handling the post-update-cmd event returned with error code 1
```

Am I doing something wrong?

I'm also noticing that for the `psr/log` of the `paypal/rest-api-sdk-php` the dependencies are moved into 
`C:\Users\kmgal\Desktop\mozart-test\src\Psr\Log\vendor\psr\log\Psr\Log` instead of
`C:\Users\kmgal\Desktop\mozart-test\src\Psr\Log\`.

Again, am I doing something wrong (apart from using Windows)?

**Comment by kmgalanakis** [comment]
I've verified (with the help of a friend) that both issues occur only in Windows :(

The same `composer.json` file performs as expected in Ubuntu and OSΧ. :(

**Comment by coenjacobs** [comment]
This is indeed fixed by #21. The `master` branch now requires intensive testing before I can push the next release out, so feel free to help testing.
